Dockerfile

```diff
@@ -1,56 +1,53 @@
 # syntax=docker/dockerfile:1.4
 
 ####################################################################################################
-## Prepare Workspace
+## Build Packages
 
-FROM node:18-alpine AS workspace
-WORKDIR /workspace
+FROM node:18-alpine AS builder
+WORKDIR /directus
+
+ENV NODE_OPTIONS=--max-old-space-size=8192
 
-COPY pnpm-lock.yaml .
 RUN corepack enable && corepack prepare pnpm@7.30.0 --activate
+
+COPY pnpm-lock.yaml .
 RUN pnpm fetch
 COPY . .
 RUN pnpm install --recursive --offline --frozen-lockfile
 
-####################################################################################################
-## Create Deployment
-
-FROM workspace AS pruned
-WORKDIR /workspace
-ENV NODE_OPTIONS=--max-old-space-size=8192
-
-RUN pnpm --recursive --workspace-concurrency=1 run build
-RUN pnpm --filter directus deploy --prod pruned
-RUN cd pruned \
+RUN : \
+	&& pnpm --recursive --workspace-concurrency=1 run build \
+	&& pnpm --filter directus deploy --prod dist \
+	&& cd dist \
 	&& pnpm pack \
 	&& tar -zxvf *.tgz package/package.json \
 	&& mv package/package.json package.json \
 	&& rm -r *.tgz package \
-	&& mkdir database extensions uploads
+	&& mkdir -p data/database data/extensions data/uploads \
```

Comment:
Suggestion: we should do everything within a single `data` volume.

Reply:
We actually did this on the first release, but had to revert as it was a breaking change as compared to the current Docker image, which relied on this structure